extensions/wikia/DesignSystem/i18n/build.js

```diff
@@ -0,0 +1,51 @@
+var fs = require('fs'),
+	path = require('path'),
+	rootDir = process.cwd() + '/node_modules/design-system-i18n/i18n',
+	filename = 'design-system.json',
+	destDir = './i18n',
+	// keep in sync with DesignSystem/DesignSystemHelper.class.php
+	messageParamsMapping = {
```

Comment:
What do you think about extracting this mapping into separate json file and re-use it in php and js?